sdl-core/src/main/scala/io/smartdatalake/workflow/connection/JdbcTableConnection.scala

```diff
@@ -109,6 +133,17 @@ case class JdbcTableConnection( override val id: ConnectionId,
     } else Map()
   }
 
+  // setup connection pool
+  val pool = new GenericObjectPool[SqlConnection](new JdbcClientPoolFactory)
+  pool.setMaxTotal(maxParallelConnections)
+  pool.setMaxIdle(1) // keep max one idle jdbc connection
```

Comment:
Maybe we could make these parameters configurable? Especially the timeout: if you have many small feeds you might want to increase the value.

Reply:
Good idea, i added connectionPoolMaxIdleTimeSec in JdbcTableConnection and SftpFileRefConnection